src/MainWindow.cpp

```diff
@@ -649,7 +656,8 @@ void MainWindow::refreshProfiles()
 #if (QT_VERSION >= QT_VERSION_CHECK(4, 4, 0))
         mUI.profileComboBox->insertSeparator(mUI.profileComboBox->count());
 #endif
-        mUI.profileComboBox->addItem(QObject::tr("(default)"), QVariant(QString::Null()));
+        // In workbench, there is always a default profile with no rules selected
+        mUI.profileComboBox->addItem(QObject::tr("(default) (0)"), QVariant(QString::Null()));
```

Comment:
This is not correct, `(default)` profile can have selected rules if any of the Rules have the `@selected` attribute set to `true`. You can use the same code with `xccdf_policy_get_selected_rules_count` to check the default profile.